packages/dev/core/src/Particles/particleSystemComponent.ts

```diff
@@ -93,6 +93,12 @@ Engine.prototype.createEffectForParticles = function (
         defines += "\n#define BILLBOARD\n";
     }
 
+    if (particleSystem?.isAnimationSheetEnabled) {
+        if (defines.indexOf(" ANIMATESHEET") === -1) {

```

Comment:
Just to be sure: the leading space is intended?

Reply:
yup it is to find #define ANIMATE... like it is done for billboard for consistency but good to ask !!!